diff --git a/core/Makefile b/core/Makefile
index 20d988a..82740c0 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -398,7 +398,7 @@ $(INSTALLED_BOOTIMAGE_TARGET): $(MKBOOTIMG) $(INTERNAL_BOOTIMAGE_FILES)
 	$(hide) $(call assert-max-image-size,$@,$(BOARD_BOOTIMAGE_PARTITION_SIZE),raw)
 endif # TARGET_BOOTIMAGE_USE_EXT2
 
-#include $(BUILD_SYSTEM)/kernel.mk
+include $(BUILD_SYSTEM)/kernel.mk
 
 else	# TARGET_NO_KERNEL
 # HACK: The top-level targets depend on the bootimage.  Not all targets
